posthog/api/insight_comment.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers, viewsets
+from rest_framework.permissions import IsAuthenticated
+
+from posthog.api.routing import StructuredViewSetMixin
+from posthog.api.shared import UserBasicSerializer
+from posthog.models import InsightComment
+
+
+class InsightCommentSerializer(serializers.ModelSerializer):
+    created_by = UserBasicSerializer(read_only=True)
+
+    class Meta:
+        model = InsightComment
+        fields = [
+            "id",
+            "insight",
+            "comment",
+            "created_at",
+            "created_by",
+        ]
+
+
+class InsightCommentsViewSet(StructuredViewSetMixin, viewsets.ModelViewSet):
+    queryset = InsightComment.objects.all()
```

Comment:
Unclear how we're guaranteeing on this model that there aren't any data leaks (i.e. that you can't see the records from another project/org to which you don't have permissions). Also, we don't have functional tests for this endpoint.